src/test/new-project.ts

```diff
@@ -361,17 +359,21 @@ describe('Create new project from prompts and command line overrides', () => {
         .on('end', done);
     });
 
-    it('creates expected files', (done) => {
-      let name = argument[1] ? argument[1] : answers.name;
+    it('creates expected files', async function() {
       let manifestFileName = 'manifest.xml';     
 
       let expected = [
         manifestFileName,
         ...expectExcelCustomFunctionFiles       
       ];
 
-      // assert.file(expected);
-      done();
+      assert.file(expected);
+    });
+    
+    it('manifest display name set to customDislayName', async function() {
+      let manifestFilePath = path.win32.resolve('manifest.xml');
+      const info =  await officeAddinManifest.readManifestFile(manifestFilePath);
+      assert.textEqual(info.displayName, customDislayName);
```

Comment:
Also you can verify that a guid was written for the manifest id and that it was changed from the original value.